schema/swagger-extensions.json

```diff
@@ -1122,6 +1142,9 @@
         "example": { },
         "x-ms-client-flatten": {
           "$ref": "#/definitions/xmsClientFlatten"
+        },
+        "x-ms-client-name": {
+          "type": "string"
```

Comment:
overall the addition of this extension definition looks good to me. Just curious to know, if there was a special reason to not have description here v/s having it in other places?